HW03.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: "HW03"
+author: "Laura Hunter"
+date: "7/19/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library("ggplot2")
+library("gridExtra")
+```
+
+## Timing Score Plots
+
+In the [neuroeconomics lab](https://www.gottlieblab.com/) where I worked as a research technician, I created a project based on [this paper](https://9fabbb78-5b5b-4a1b-aeb0-b5443b6eac2f.filesusr.com/ugd/9cf124_28146e8d3fa5431baaa796c288f0724c.pdf). It involves two lotteries, wherein the participant needs to make a choice between one that has a high level of variability in reward size (e.g. you win 100 points or 200 points) and one that has a low level of variability in reward size (e.g. you win 140 points or 160 points).
+
+The details of the experimental design are not important for understanding the timing scores, however. Once our data collection (ongoing) is complete, we expect about 200 participants, so I am scoring each participants' performance in order to remove those that clearly misunderstood the task.
+
+The participants had a minimum of 600 milliseconds and a maximum of 10 seconds to make a decision. In the dataset I'm loading here, I have calculated each participant's ability to do this task within that timing window (number of trials with bad timing over total number of trials). Once I did that, I Z-Scored them to get a feel for what we should expect from participants. This is in the hopes of knowing which participants were "too bad" at the task (too positive/far to the right) and therefore need to be removed from our datapool.
+
```

Comment:
The explanation of the background is clear and sufficient. And the project sounds very interesting, thanks for taking the time on explaining it with this detail :)